pySDC/playgrounds/Diagonal/optimDiagCoeffs.md

```diff
@@ -0,0 +1,57 @@
+|   M | quadType    | distr    | optim   | coeffs                                             |         rho |
+|----:|:------------|:---------|:--------|:---------------------------------------------------|------------:|
+|   2 | GAUSS       | LEGENDRE | QmQd    | (0.105662, 0.394338)                               | 4.1514e-10  |
+|   2 | GAUSS       | LEGENDRE | QmQd    | (0.394338, 0.105662)                               | 3.06546e-10 |
+|   2 | GAUSS       | LEGENDRE | Speck   | (0.166667, 0.5)                                    | 1.44955e-10 |
+|   2 | GAUSS       | LEGENDRE | Speck   | (0.5, 0.166667)                                    | 2.51252e-10 |
```

Comment:
Is there an intuitive explanation for why you can swap the values on the diagonal and get about the same spectral radius? The full Q matrix is probably not really symmetric in a similar way?

Reply:
No the full Q matrix is not symmetric indeed :

```python
>>> genCollocation(2, 'LEGENDRE', 'GAUSS')[-1]
array([[ 0.25      , -0.03867513],
       [ 0.53867513,  0.25      ]])
```

and I did not find any intuitive reason to explain this. I guess at this point this should be investigated by hand ...